File: Week8/oddrange.py
def OddRange(num1,num):
	l=[]
	for i in range(num1,num2):
		if(i%2!=0):
			l.append(i)
	return(l)
num1=int(input("Enter num1 :"))
num2=int(input("Enter num2 :"))	
print("The odd numbers are : ",OddRange(num1,num2))			

File: Week8/oddrange.js


document.getElementById("demo").innerHTML = "Paragraph changed.";
var num1=parseInt(prompt("Enter num1 :"));
var num2=parseInt(prompt("Enter num2 :"));

function oddRange(num1,num2)
{
	
	for(var i=num1;i<=num2;i++)
	{
		if(i%2!=0)
			document.write(i);

			console.log("debug1");
	}
	console.log("debug2");

	

}
oddRange(num1,num2);			
